test/spectests/spectests.cpp

```diff
@@ -98,12 +101,12 @@ class test_runner
         {
             const auto type = cmd.at("type").get<std::string>();
 
-            log_no_newline("Line " + std::to_string(cmd.at("line").get<int>()) + ": " + type + " ");
+            m_current_line = cmd.at("line").get<int>();
+            m_current_test_type = type;
 
             if (type == "module")
             {
                 const auto filename = cmd.at("filename").get<std::string>();
-                log_no_newline("Instantiating " + filename + " ");
```

Comment:
Removing this, as I don't think it was very useful, and it would complicate things a bit to keep it.
The line will read just `Line X: module PASSED`